docs/json/sonarr/cf/french-voq.json

```diff
@@ -0,0 +1,17 @@
+{
+  "trash_id": "802dd70b856c423a9b0cb7f34ac42be1",
```

Comment:
are there supposed to be no scores? if so, looks good to me 😄

Reply:
Yes, both are informative. It is like all the other French audio CFs. They exist just for people to know which French they will potentially get 😅